biogeochem/EDPhysiologyMod.F90

```diff
@@ -1853,8 +1854,10 @@ subroutine recruitment( currentSite, currentPatch, bc_in )
 
 
     do ft = 1,numpft
-       if(currentSite%use_this_pft(ft).eq.itrue)then
-       temp_cohort%canopy_trim = init_recruit_trim
+       if(currentSite%use_this_pft(ft).eq.itrue &
+            .and. ((hlm_use_nocomp .eq. ifalse) .or. (ft .eq. currentPatch%nocomp_pft_label)))then
+
+          temp_cohort%canopy_trim = init_recruit_trim
```

Comment:
I guess the crux of this is that the use_this_pft flag is a site level and not patch level quantity.  I wonder 
a) whether use_this_pft should be a patch level quantity. 
b) if not, could we put a comment where it is calculated (or here) to explain why this patch-level logic is required in nocomp mode?

Reply:
@rosiealice I added the comment you requested above.